sample-apps/blank-java/src/main/java/example/Handler.java

```diff
@@ -1,65 +1,43 @@
-import java.util.Map;
-import java.util.List;
-import java.util.concurrent.CompletableFuture;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.lambda.LambdaClient;
+import software.amazon.awssdk.services.lambda.model.GetAccountSettingsResponse;
+import software.amazon.awssdk.services.lambda.model.LambdaException;
 
 // Handler value: example.Handler
-public class Handler implements RequestHandler<SQSEvent, String>{
-  private static final Logger logger = LoggerFactory.getLogger(Handler.class);
-  private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
-  private static final LambdaAsyncClient lambdaClient = LambdaAsyncClient.create();
-  public Handler(){
-    CompletableFuture<GetAccountSettingsResponse> accountSettings = lambdaClient.getAccountSettings(GetAccountSettingsRequest.builder().build());
-    try {
-      GetAccountSettingsResponse settings = accountSettings.get();
-    } catch(Exception e) {
-      e.getStackTrace();
-    }
-  }
-  @Override
-  public String handleRequest(SQSEvent event, Context context)
-  {
-    String response = new String();
-    // call Lambda API
-    logger.info("Getting account settings");
-    CompletableFuture<GetAccountSettingsResponse> accountSettings = 
-        lambdaClient.getAccountSettings(GetAccountSettingsRequest.builder().build());
-    // log execution details
-    logger.info("ENVIRONMENT VARIABLES: {}", gson.toJson(System.getenv()));
-    logger.info("CONTEXT: {}", gson.toJson(context));
-    logger.info("EVENT: {}", gson.toJson(event));
-    // process event
-    for(SQSMessage msg : event.getRecords()){
-      logger.info(msg.getBody());
-    }
-    // process Lambda API response
-    try {
-      GetAccountSettingsResponse settings = accountSettings.get();
-      response = gson.toJson(settings.accountUsage());
-      logger.info("Account usage: {}", response);
-    } catch(Exception e) {
-      e.getStackTrace();
+public class Handler implements RequestHandler<Map<String,String>, String> {
+
+    private static final Logger logger = LoggerFactory.getLogger(Handler.class);
+    private static final Region region = Region.US_EAST_1;
+    private static final LambdaClient awsLambda = LambdaClient.builder()
+        .region(region)
+        .build();
+
+    @Override
+    public String handleRequest(Map<String,String> event, Context context) {
+        
+        // log execution details
+        logger.info("ENVIRONMENT VARIABLES: " + System.getenv());
+        logger.info("EVENT: " + event);
+
+        // process event
+        GetAccountSettingsResponse response;
+        try {
+            response = awsLambda.getAccountSettings();
+            System.out.println("Total code size for your account is " + response.accountLimit().totalCodeSize() + " bytes");
+        } catch(LambdaException e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
+        return "200 OK";
```

Comment:
I think this might be confusing to people. It might give the impression that this is how to generate a HTTP status 200 response with a body of "OK".

Reply:
Should I return the totalCodeSize() result instead? i.e.

`return response != null ? "Total code size for your account is " + response.accountLimit().totalCodeSize() + " bytes" : "Error";`

Reply:
Yes that makes sense and gives the function a purpose.